app/main/views.py

```diff
@@ -125,7 +127,7 @@ def get_associations(resource_id):
             values = set(curr)
```

Comment:
The previous line should be changed to curr['value'].append(val) since now "curr" is an object not just a list!